scripts/check_compliance.py

```diff
@@ -165,6 +167,23 @@ def run(self):
         # single Kconfig tree and warns for all references to undefined symbols
         os.environ["KCONFIG_STRICT"] = "y"
 
+        kconfig_env_var_list_path = os.path.join(self.zephyr_base,
+                                    os.path.normpath("cmake/kconfig_env_vars.txt"))
+        # Skipping last new line which gives empty string
+        kconfig_env_var_list = open(kconfig_env_var_list_path).read().split("\n")[:-1]
```

Comment:
Can use `readlines()`.

Reply:
but doesn't `readlines()` append the `\n` to the string?

Reply:
Ah, right, had forgotten that. `.read().splitlines()` seems to work.

Reply:
yeah, that's better than manually trying to split for `\n`

Reply:
I'll add that instead

Reply:
Done